idd/Energy+.idd.in

```diff
@@ -51906,6 +51982,10 @@ AirLoopHVAC:UnitarySystem,
        \required-field
        \type alpha
        \note Unique name for the Unitary System.
+       \reference-class-name validBranchEquipmentTypes
+       \reference validBranchEquipmentNames
+       \reference-class-name validBranchEquipmentTypes
```

Comment:
@mbadams5 I will update the idd, merge in develop and will push the branch. Are you are not working on it?